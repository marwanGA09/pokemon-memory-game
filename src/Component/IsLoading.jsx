export function IsLoading() {
  return <img src="Spinner.gif" alt="spinner gif image" />;
}
