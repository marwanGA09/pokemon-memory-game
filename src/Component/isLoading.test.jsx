import { render, screen } from '@testing-library/react';
import { describe, expect, test } from 'vitest';
import { IsLoading } from './IsLoading';

describe('isLoading test', () => {
  test('found in the document', () => {
    render(<IsLoading />);
    const imageComponent = screen.getByRole('img');
    // screen.debug(imageComponent);
    expect(imageComponent).toBeInTheDocument();
  });

  test('Take snap shot', () => {
    render(<IsLoading />);
    expect(screen.getByRole('img')).toMatchSnapshot();
  });
});
