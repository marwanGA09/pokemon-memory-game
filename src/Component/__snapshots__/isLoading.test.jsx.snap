// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`isLoading test > Take snap shot 1`] = `
<img
  alt="spinner gif image"
  src="Spinner.gif"
/>
`;
